example/example.go

```diff
@@ -15,7 +15,7 @@ func main() {
 		{Count: 4, Color: graph.Color4},
 	}
 
-	g := graph.NewGraph(contribs)
+	g := graph.NewGraph(contribs, 48)
```

Comment:
Let's pass an options struct with a padding option instead, and have the pillars always scale if above a hardcoded threshold, and the padding should start from the highest point from the pillars.